SIG.md

```diff
@@ -0,0 +1,85 @@
+# Cilium SIGS
+
+The Cilium project has Special Interest Groups, or SIGs, with a common purpose of advancing the project with respect to a specific topic, such as network policy or documentation. Our goal is to enable a distributed decision structure and code ownership, as well as providing focused forums for getting work done, making decisions, and onboarding new contributors. 
+
+Areas covered by SIGs may be vertically focused on particular components or functions, cross-cutting/horizontal, spanning many/all functional areas of the project, or in support of the project itself. Examples:
+
+- Policy
+- Scalability
+- Docs
+
+Each SIG must have a charter that specifies its scope (topics, subsystems, code repos, and/or directories), responsibilities, and areas of authority. See the SIG charter process for details on how charters are managed. SIGs are relatively free to customize or change how they operate, within some broad guidelines and constraints imposed by cross-project processes (e.g., the release process) and assets (e.g., the Cilium repo).
+
+SIGs are designed to help contributors gain depth in a specific area of the Cilium project. Committers that are aspiring reviewers of the Cilium project can gain expertise by getting involved with a SIG and getting assigned PRs.
+
+SIGs don't have any additional authority - they are expected to drive towards solutions that can be accepted by the project's committer community - ideally by consensus, and if necessary, a vote. 
+
+A primary reason that SIGs exist is as forums for collaboration. Much work in a SIG should stay local within that SIG. However, SIGs must communicate in the open, ensure other SIGs and community members can find notes of meetings, discussions, designs, and decisions, and periodically communicate a high-level summary of the SIG's work to the community.
+
+# SIG Roles
+
+## SIG Lead
+
+- Membership tracked in [sigs.yaml](./sigs.yaml)
+- MUST be a reviewer of one or more areas of the project, and should be a subject matter expert in their SIG area.
+- SHOULD drive charter changes (including creation) to get community buy-in but MAY delegate content creation to SIG contributors.
+- MUST organize a method of keeping cadence (meetings, async channel in Cilium Slack, etc.), and make sure [sigs.yaml](./sigs.yaml) is up to date.
+- SHOULD facilitate cadence but MAY delegate to other members or future Leads/Leads in training.
+- MUST ensure notes made available.
+- MAY delegate the actual communication and creation of content to other contributors where appropriate.
+
+### Requirements
+Leads MUST be at least a "reviewer" on our [contributor ladder](https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md#reviewer) to be eligible to hold a leadership role within a SIG.
+SIGs MAY prefer various levels of domain knowledge depending on the role. This should be documented.
```

Comment:
Sorry, I don't understand this sentence.

Reply:
It's meant as leads can have different areas of expertise that's applicable to that sig, for example if sig-servicemesh had a lead in security, one in networking, etc

Reply:
OK I get it, thank you. Can we rephrase it, please? Maybe something like:

```
Individual Leads MAY focus on specific subdomains for the SIG, such as security or networking in the context of a given SIG's scope. If some Leads chose to focus on particular domains of knowledge inside of the SIG, this SHOULD be documented.
```

Reply:
This comment is marked as resolved, but has not been addressed

Reply:
oh my bad - sorry about that! Thanks Quentin 🥹